packages/react-components/src/components/utils/RichTextEditorUtils.ts

```diff
@@ -180,6 +180,9 @@ export const removeLocalBlobs = (
   removedInlineImages: Record<string, string>[]
 ): void => {
   removedInlineImages.forEach((image) => {
+    if (!image.id) {
```

Comment:
@Leah-Xia-Microsoft please review this closely to make sure this is how we would expect this to be handled if image.id does not exist

Reply:
Could we have a console error log here saying that Failed to remove the image from the local blob: Image ID is missing or invalid?